src/core/binding.h

```diff
@@ -383,30 +394,24 @@ _IRQL_requires_max_(DISPATCH_LEVEL)
 void
 QuicBindingRemoveSourceConnectionID(
     _In_ QUIC_BINDING* Binding,
-    _In_ QUIC_CID_HASH_ENTRY* SourceCid,
-    _In_ CXPLAT_SLIST_ENTRY** Entry
+    _In_ QUIC_CID_HASH_ENTRY* SourceCid
     );
 
 //
-// Removes all the connection's source CIDs from the binding's lookup table.
+// Removes all the source CIDs from the binding's lookup table.
 //
 _IRQL_requires_max_(DISPATCH_LEVEL)
 void
-QuicBindingRemoveConnection(
+QuicBindingRemoveAllSourceConnectionIDs(
     _In_ QUIC_BINDING* Binding,
     _In_ QUIC_CONNECTION* Connection
     );
 
-//
-// Moves all the connections source CIDs from the one binding's lookup table to
-// another.
-//
 _IRQL_requires_max_(DISPATCH_LEVEL)
 void
-QuicBindingMoveSourceConnectionIDs(
-    _In_ QUIC_BINDING* BindingSrc,
-    _In_ QUIC_BINDING* BindingDest,
-    _In_ QUIC_CONNECTION* Connection
+QuicBindingRemoveRemoteHash(
```

Comment:
Does this mean there's no way to move source CIDs between bindings anymore?

Reply:
No. We can move source CIDs between old binding and new binding by setting QUIC_PARAM_CONN_LOCAL_ADDRESS. In this case, we will call QuicBindingAddAllSourceConnectionIDs() for new binding and QuicBindingRemoveAllSourceConnectionIDs() for old binding.